app/views/users/profile.html.erb

```diff
@@ -280,6 +280,17 @@
   ul.nav-tabs>li a{
     padding: 10px;
   }
+  #link_social_media{
```

Comment:
hi @Paarmita, the CSS is implemented on button and changes are expected on profile icons so some changes would be required.